Darkest Crusade/Assets/core/GameState/GameState.cs

```diff
@@ -8,6 +8,6 @@ namespace Core
   public class GameState : Singleton<GameState>
   {
     public List<GameObject> heroes;
+    public DungeonState DungeonState;
```

Comment:
It would be better to use `ReactiveProperty<DungeonState>`.

Reply:
This field will be initialized once and will not change. Why he should use reactive property ?